### PR TITLE
fix: update design to match Line Listing app

### PR DESCRIPTION
**Pull request by cooper-joe**
Implements [UX-130](https://dhis2.atlassian.net/browse/UX-130)

---

### Description

Updates Data Visualizer color, spacing, and typography usage to better match the updated styles defined by Line Listing app. 

---

### Screenshots

Before:

image

After:

image

[UX-130]: https://dhis2.atlassian.net/browse/UX-130?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/DimensionsPanel/DndDimensionsPanel.js [hunk 1]
```diff
@@ -19,15 +19,20 @@ export class DndDimensionsPanel extends Component {
...
-                    text={this.state.filterText}
-                    onChange={this.onFilterTextChange}
-                    onClear={this.onClearFilter}
-                    disableUnderline={true}
-                    type="search"
-                    dataTest="dimensions-panel-filter"
-                />
+                <div
+                    className={styles.filter}
+                    data-test="dimensions-panel-filter"
```
Comment (martinkrulltott) [inline src/components/DimensionsPanel/DndDimensionsPanel.js:24]:
```suggestion
```
`dimensions-panel-filter` is already passed to `DimensionFilter` below so this shouldn't be set on the `div` as well.

Comment (martinkrulltott) [reply src/components/DimensionsPanel/DndDimensionsPanel.js:24]: For reference, `dataTest` / `data-test` is only there for our Cypress tests to find elements

**Review by martinkrulltott — APPROVED** [review]
The code LGTM, except for a redundant `data-test` prop that should be removed.

**Comment by cooper-joe** [comment]
> The code LGTM, except for a redundant data-test prop that should be removed.

Removing that `data-test` attribute seems to have caused 390 cypress tests to fail. Before that change, everything passed. Any idea what broke?

**Pull request merged**